pkg/lockfile/rpmdb_test.go

```diff
@@ -0,0 +1,64 @@
+package lockfile_test
+
+import (
+	"testing"
+
+	"github.com/google/osv-scanner/pkg/lockfile"
+)
+
+// Berkeley DB (rpm < v4.16)
+func TestParseRpmDB_BDB_Single(t *testing.T) {
+	t.Parallel()
+
+	packages, err := lockfile.ParseRpmDB("fixtures/rpm/Packages")
+
+	if err != nil {
+		t.Errorf("Got unexpected error: %v", err)
+	}
+
+	expectPackages(t, packages, []lockfile.PackageDetails{
+		{
+			Name:      "htop",
+			Version:   "3.2.1",
+			Ecosystem: lockfile.RedHatEcosystem,
+			CompareAs: lockfile.RedHatEcosystem,
+		},
+	})
+}
+
+func TestParseRpmDB_SQLite_Single(t *testing.T) {
+	t.Parallel()
+
+	packages, err := lockfile.ParseRpmDB("fixtures/rpm/rpmdb.sqlite")
+
+	if err != nil {
+		t.Errorf("Got unexpected error: %v", err)
+	}
+
+	expectPackages(t, packages, []lockfile.PackageDetails{
+		{
+			Name:      "htop",
+			Version:   "3.2.1",
+			Ecosystem: lockfile.RedHatEcosystem,
+			CompareAs: lockfile.RedHatEcosystem,
+		},
+	})
+}
+
+func TestParseRpmDB_SQLite_FileDoesNotExist(t *testing.T) {
```

Comment:
nit: I typically had these kind of checks at the top as they're usually pretty short and test the first lines of the parser - while by itself not a big deal, I think we should still move these to be consistent with the existing tests